content/machine/src/main/java/net/qbar/common/tile/creative/TileCreativeWaterGenerator.java

```diff
@@ -0,0 +1,119 @@
+package net.qbar.common.tile.creative;
+
+import java.util.EnumMap;
+import java.util.List;
+
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ITickable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fluids.FluidRegistry;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.FluidTank;
+import net.minecraftforge.fluids.capability.CapabilityFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.qbar.common.fluid.CreativeFluidTank;
+import net.qbar.common.tile.TileInventoryBase;
+
+public class TileCreativeWaterGenerator extends TileInventoryBase implements ITickable
+{
+    private final FluidTank fluidTank;
+    private final int transferCapacity;
+    EnumMap<EnumFacing, IFluidHandler> handler;
```

Comment:
Need private and final modifiers
Blank line needed between vars and constructor